### DISCUSSION TITLE
Keyboard zones got selected instead of per-key

### DISCUSSION THREAD

**Discussion by gingusScringus** [T-Troll/alienfx-tools#386] [Q&A] [answered]
This might be a stupid thing to post, because i might missed something as I'm fairly new to this software. I have a Dell G5 5500 and i wanted to try out this program due to how crap the AWCC app is. It broke after an update for the software, and I couldn't use the FX tab anymore.

Anyway. I did some reading on how to use this program, and although the program did detect my laptop when clicking on "Detect devices". I wanted to manually create it so that i could make it per-key. But something I noticed, when I shuffled through the lights using the shortcut key. The keyboard portion got selected multiple keys (or selected the zones), despite i already cleared out the configurations beforehand.

I don't know it this is hardcoded to the keyboard or something, I expected that when i shuffle through the lights. Each keys get shuffled, not the zones or multiple keys. I also ran `./alienfx-cli status` on PowerShell and here's what I got:
![image](https://github.com/T-Troll/alienfx-tools/assets/67906219/8541a4cd-0ddf-4618-99cb-3e3a4fa37287)

and here's what my AlienFX Control configuration right now:
![image](https://github.com/T-Troll/alienfx-tools/assets/67906219/4de508ae-efcc-440f-9ebb-ca737d6449d3)
This is just the Light bar right now, I haven't configured the keyboard part yet. So if anyone are able to help me with this, I'd appreciate it. Thanks.

Side note: I sometimes couldn't enable/disable the keyboard lights with the F10 key. I'm not actually sure how the hell was I able to turn on the keyboard. I couldn't get it to turn on before (mostly because i didn't really understand the program), and also, the keyboard light didn't reset after restart. So I don't know.

**Comment by T-Troll** [comment, marked as answer]
5500 have light bar and (optionally) 4-zone keyboard.
Yes, not per-key, but 4 zones only (quite common for cheaper Alienware).
Seems like you light scheme doesn't fit your hardware exactly, try to use manual setup.

NB: "Devices" tab used to DEFINE the lights. "Colors/etc" tabs are used to set it for different functions.

**Reply by gingusScringus** [reply]
Oh, well that's sad. Was hoping to have a key get lit when pressed using the program, guess have to make the entire zone lit up if i wanna do that. Thanks for the help though.
